### PR TITLE
created mt_service_api

### PR DESCRIPTION
**Pull request by MSurvonen**

### CODE REVIEW

**Review by bunjovis — CHANGES_REQUESTED** [review]
the files should be moved to cc-mt-service folder. Otherwise looking good. One common comment to everyone doing these API services: be careful when using generated code, you still have to know what the code is doing, how to use it or modify it.

File: cc-mt-service/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+              $ref: "#/components/schemas/Advert"
+        description: advert data
+        required: true
+      responses:
+        "200":
+          description: advert added
+          content:
+            application/json:
+              schema:
+                $ref: "#/v1/components/schemas/Added"
```
Comment (OpaJaa) [inline cc-mt-service/openapi.yaml]: Remove /v1/

File: cc-mt-service/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        "400":
+          description: adding failure
+          content:
+            application/json:
+              schema:
+                $ref: "#/v1/components/schemas/Error"
```
Comment (OpaJaa) [inline cc-mt-service/openapi.yaml]: Remove /v1/

File: cc-mt-service/openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                    cityId:
+                      type: string
+                    region:
+                      type: string
+                    regionId:
+                      type: string
+                    countryCode:
+                      type: string
+            attachments:
+              type: array
```
Comment (OpaJaa) [inline cc-mt-service/openapi.yaml:123]: array needs items. https://swagger.io/docs/specification/data-models/data-types/#array

File: cc-mt-service/openapi.yaml [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            regions:
+              type: array
```
Comment (OpaJaa) [inline cc-mt-service/openapi.yaml:136]: Same here.

**Review by OpaJaa — CHANGES_REQUESTED** [review]

File: cc-mt-service/openapi.yaml [hunk 5]
```diff
@@ -0,0 +1,152 @@
+openapi: '3.0.2'
```
Comment (hannakortetjarvi) [inline cc-mt-service/openapi.yaml]: backend and pk service design documents are using openapi 3.0.3, should all use the same version?

**Review by OpaJaa — DISMISSED** [review]

**Review by rvsape — DISMISSED** [review]

**Review by hannakortetjarvi — APPROVED** [review]

**Review by bunjovis — APPROVED** [review]

**Review by OpaJaa — APPROVED** [review]

**Pull request merged**